### DISCUSSION TITLE
How to use archivebox to download wiki.archlinux.org

### DISCUSSION THREAD

**Discussion by Abraka** [ArchiveBox/ArchiveBox#832] [General]
Hello, how to use a archivebox to download wiki pages as https://wiki.archlinux.org/ to get some useful local archiv. If I do snapshot of https://wiki.archlinux.org/ with depth 1 it results in mismatch of different sites istead of of wiki with subpages. Whats the right way to do this?

**Comment by pirate** [comment]
See https://github.com/ArchiveBox/ArchiveBox/issues/191

**Comment by Abraka** [comment]
As few others I did expect that complex archive of website is the basic feature of sw as archivebox. But it seems you working on it, so I will wait fot it. Anyway thx for your work and time.
